### PR TITLE
Update

### PR DESCRIPTION
**Pull request by hinammehra**

### CODE REVIEW

File: lib/xeroizer/models/allocation.rb [hunk 1]
```diff
@@ -5,9 +5,19 @@ class AllocationModel < BaseModel
 
     class Allocation < Base
       decimal :applied_amount
+      datetime :date
```
Comment (CloCkWeRX) [inline lib/xeroizer/models/allocation.rb]: Should these changes be in a different PR?

File: lib/xeroizer/models/overpayment.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
+module Xeroizer
```
Comment (CloCkWeRX) [inline lib/xeroizer/models/overpayment.rb]: Can you add specs for this?

https://developer.xero.com/documentation/api/overpayments/ has a fair amount of detail on what is valid and what isn't

**Comment by hinammehra** [comment]
Update
